develop/blockchains/custom-blockchains/add-smart-contract-functionality.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Add Smart Contract Functionality
+description: Add smart contract capabilities to your Polkadot SDK-based blockchain. Explore EVM and Wasm integration for enhanced chain functionality.
+---
+
+# Add Smart Contract Functionality
+
+## Introduction
+
+When building your custom blockchain with the Polkadot SDK, you have the flexibility to add smart contract capabilities through specialized pallets. These pallets allow your blockchain users to deploy and execute smart contracts, enhancing your chain's functionality and programmability.
+
+Polkadot SDK-based blockchains support two distinct smart contract execution environments: EVM (Ethereum Virtual Machine) and Wasm (WebAssembly). Each environment allows developers to deploy and execute different types of smart contracts, providing flexibility in choosing the most suitable solution for their needs.
+
+## EVM Smart Contracts
+
+To enable Ethereum-compatible smart contracts in your blockchain, you'll need to integrate [Frontier](https://github.com/polkadot-evm/frontier){target=\_blank}, the Ethereum compatibility layer for Polkadot SDK-based chains. This requires adding two essential pallets to your runtime:
+
+- [pallet-evm](https://github.com/polkadot-evm/frontier/tree/master/frame/evm){target=\_blank} - provides the EVM execution environment
+- [pallet-ethereum](https://github.com/polkadot-evm/frontier/tree/master/frame/ethereum){target=\_blank} - handles Ethereum-formatted transactions and RPC capabilities
+
+For step-by-step guidance on adding these pallets to your runtime, refer to [Add a Pallet to the Runtime](){target=\_blank}.
+
+## Wasm Smart Contracts
+
+To support Wasm-based smart contracts, you'll need to integrate:
+
+- [pallet-contracts](https://docs.rs/pallet-contracts/latest/pallet_contracts/index.html#contracts-pallet){target=\_blank} - provides the Wasm smart contract execution environment
+
+This pallet enables the deployment and execution of Wasm-based smart contracts on your blockchain. For detailed instructions on adding this pallet to your runtime, see [Add a Pallet to the Runtime](){target=\_blank}.
+
+## Where to Go Next
+
+Now that you understand how to enable smart contract functionality in your blockchain, you might want to:
+
+- Learn how to develop and deploy smart contracts by visiting the [Smart Contracts](/develop/smart-contracts/overview){target=\_blank} guide
```

Comment:
I agree on linking the overview page at the end of this page, but since the whole page is short, I'd add more links, maybe to native evm contracts or Parachain contracts as well